manifests/pipelines/git-fetch-pipeline.yaml

```diff
@@ -258,7 +214,7 @@ spec:
           labels:
             app: $(params.model-name)-$(params.model-version)
             model-name: $(params.model-name)
-          name: "$(tasks.sanitise-model-name.results.output-string)-$(params.model-version)"
+          name: $(params.model-name)-$(params.model-version)
```

Comment:
This is one example but maybe because it was a filename change this didn't get caught in the rebase but these need to be the `tasks.sanitise-model-name.results.output-string`.

It should be a quick fix any just match where they are in the `s3-fetch` pipeline